Soft delete (#8999)
camunda/camunda-docs: docs/components/hub/workspace/manage-projects/recently-deleted.md

---
id: recently-deleted
title: Recover deleted files

> description: Recover deleted files within 30 days of deletion.

Reviewer: I try to keep descriptions around 160 characters for SEO purposes but this is nit-picky, and you can disregard.

---

> Learn how to recover recently-deleted files before they're permanently removed.

Reviewer: ```suggestion
Learn how to recover recently deleted files before they're permanently removed.
```

adverbs ending in -ly are not hyphenated with the following participle/adjective in standard style, but I think you could technically go either way here?

Author: Thanks, fixed!

## Soft deletion in Camunda Hub

> When you delete a file, it's removed from the project and moved to **Recently deleted**. You have 30 days to restore it before it's permanently deleted.

Reviewer: ```suggestion
When you delete a file, it's removed from the project and moved to **Recently deleted**. You have 30 days to restore it before permanent deletion.
```

:::note

Soft deletion only applies to files deleted in Camunda 8.10 and later. All items deleted in earlier versions are permanently deleted and can't be recovered.

:::

## Permanent deletion in Camunda Hub

Permanent deletion occurs 30 days after a file is deleted. This removes all associated data, including file content, version history, metadata, and its Git links.

## Restore permissions

> Only users with project-level **edit** or **admin** access at the time of the attempted restore, may restore a recently-deleted file. The role at the time of the original deletion is not considered.

Reviewer: ```suggestion
Only users with project-level **edit** or **admin** access at the time of the restore attempt can restore a recently deleted file. The role at the time of the original deletion is not considered.
```

> ## Browse recently-deleted files

Reviewer: ```suggestion
## Browse recently deleted files
```

> In Camunda Hub, in the left-hand navigation, click **Recently deleted**. This shows all files deleted from any projects within the last 30 days.

Reviewer: ```suggestion
In Camunda Hub, in the left navigation, click **Recently deleted**. This page lists files deleted within the last 30 days.
```

| **Name**           | The file name                                                                                                                                               |
| **Type**           | The type of deleted resource                                                                                                                                |
| **Location**       | The full path of the file's current live ancestors as a breadcrumb. Composed at read time, so renames or moves of live ancestors are reflected immediately. |
| **Deleted by**     | The user who performed the deletion                                                                                                                         |
| **Deleted on**     | The date and time of the deletion                                                                                                                           |
| **Days remaining** | Days left before permanent deletion                                                                                                                         |

> The default sort order has the most recently-deleted files first.

Reviewer: ```suggestion
By default, the list is sorted with the most recently deleted files first.
```
